### ISSUE TITLE
Forme des citations 

### ISSUE THREAD

**Issue by Kenzi-Antonin** [heig-vd-tin/template-tb#8]
Si j'ai bien compris les règles écoles disponibles sur intranet pour les citations : 
https://intra.heig-vd.ch/departement/departement-tin/etudiants/Pages/tb_collab.aspx
 
Les citations doivent être numérique et trié par ordre d'apparition.

Pour cela, il faudrait modifier les ligne 238 à 234 par :

```latex
\RequirePackage[
  backref=true,
  backend=biber,
  style=numeric,
  sorting=none,
]{biblatex}
```

**Comment by yves-chevallier** [comment]
Non les règles de la HEIG-VD n'imposent pas un format spécifique pour les références aux citations. La forme numérique est simple, et c'est celle par défaut dans Word. Il est néanmoins préférable d'utiliser un mode de citation plus mémo technique dans un rapport de plus de 10 pages. 

N'êtes vous pas satisfait avec les citations actuelles ?

**Comment by Kenzi-Antonin** [comment]
Je trouve qu'il serait cohérent de respecter le format décrit dans le document présenté par l'école.

**Comment by yves-chevallier** [comment]
Je suis d'accord avec vous qu'il devrait y avoir une cohérence entre les règles école et le template proposé. J'ai passablement discuté de ces règles avec les principaux auteurs qui n'ont pas réellement d'argumentaire solide. J'ai alors adopté des conventions d'édition basé sur des consensus plus généraux. En outre il faut noter que les conventions auxquels vous faites référence sont différents pour TIC, TIN et les autres départements.

Votre remarque est néanmoins pertinente et je tacherais d'en tenir compte la prochaine fois que je discuterai des règles des TB avec les chefs de département.
________________________________
